spec.md

```diff
@@ -583,6 +584,22 @@ message VolumeInfo {
 }
 ```
 
+##### CreateVolume Errors
+
+If the plugin is unable to complete the CreateVolume call successfully, it MUST return a non-ok gRPC code in the gRPC status.
+If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code.
+The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
+
+Condition | gRPC Code | Description | Recovery Behavior
+| --- | --- | --- | --- |
+| Volume already exists | 0 OK | Indicates that a volume corresponding to the specified volume `name` already exists. | Caller MUST assume the `CreateVolume` call succeeded. |
+| Invalid `name` | 3 INVALID_ARGUMENT | Indicates that the specified volume name is not allowed by the Plugin. More human-readable information MAY be provided in the gRPC `status.message` field. | Caller MUST fix the name before retrying. |
```

Comment:
My suggestion is to have a general "Invalid or unsupported field in the request" condition (`INVALID_ARGUMENT`), and put that in the general error section above so that we don't have to mention them here for each call. What do you think?

Reply:
I'd prefer to leave them split up so that if we want we can convert these in to detailed error codes in the future.

Reply:
One second thought, I'll go ahead and change it as you described. We can always re-add these in the future, if needed.